packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -32,6 +33,16 @@ export interface Options {
    * Cause the bootstrap peer tag to be removed after this number of ms (default: 2 minutes)
    */
   tagTTL?: number;
+  /**
+   * The interval between queries to a peer (default: 10 seconds)
+   * The interval will increase by a factor of an incrementing number (starting at 1)
+   * until it reaches the maximum attempts before backoff
+   */
+  queryInterval?: number;
+  /**
+   * The number of attempts before the queries to a peer are backed off (default: 3)
```

Comment:
```suggestion
   * The number of attempts before the queries to a peer are backed off (default: 3)
```
Before they are backed off or before they are aborted? comment and variable name have two different meanings.